### PR TITLE
add README storage

### PR DESCRIPTION
**Pull request by en-agraur**

### CODE REVIEW

File: SmartCredentials_aOS/storage/README.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+```
+
+**Non-sensitive & encrypted**
+```
+ItemContext itemContext = ItemContextFactory.createEncryptedNonSensitiveItemContext(itemType);
+```
+
+**Non-sensitive & non-encrypted**
+```
+ItemContext itemContext = ItemContextFactory.createNonEncryptedNonSensitiveItemContext(ITEM_TYPE)
```
Comment (gabrielblaj07) [inline SmartCredentials_aOS/storage/README.md:64]: itemType syntax is different from the other ones

**Review by gabrielblaj07 — CHANGES_REQUESTED** [review]

File: SmartCredentials_aOS/storage/README.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+``` 
+SmartCredentialsStorageFactory.initSmartCredentialsStorageModule(this, coreApi, securityApi);
+````
+
+### Add item to storage
+
+**Step 1:** Create item context
+
+The item context defines the item as being sensitive or non-sensitive. The sensitive repository use Android's ```SharedPreferences``` as the storage location and the non-sensitive repository use a database as the storage location. 
+The sensitive repository is recommended for small items and operations on it does not need to be performed on a background thread, while the sensitive repository is recommended for large items and operations on it must be made on a background thread.
```
Comment (gabrielblaj07) [inline SmartCredentials_aOS/storage/README.md:41]: ..while the **non-sensitive** repository is recommended for large items and operations on it must be made on a background thread.

File: SmartCredentials_aOS/storage/README.md (hunk 1 above)
Comment (en-agraur) [reply SmartCredentials_aOS/storage/README.md]: renamed to `itemType`

File: SmartCredentials_aOS/storage/README.md (hunk 2 above)
Comment (en-agraur) [reply SmartCredentials_aOS/storage/README.md]: whooops ... fixed now

**Pull request merged**